terraform/environment/locals.tf

```diff
@@ -1,4 +1,9 @@
 locals {
+  account_name = lookup(var.account_mapping, terraform.workspace, "development")
+  environment  = lower(terraform.workspace)
+
+  dns_namespace_acc = local.environment == "production" ? "" : "${local.account_name}."
```

Comment:
is that a typo mate - did you want `dns_namespace_env` ?

Reply:
It is mate, thanks!